cmd/kube-controller-manager/app/plugins.go

```diff
@@ -31,13 +31,25 @@ import (
 )
 
 // ProbeRecyclableVolumePlugins collects all persistent volume plugins into an easy to use list.
-func ProbeRecyclableVolumePlugins() []volume.VolumePlugin {
+func ProbeRecyclableVolumePlugins(config VolumeConfig) []volume.VolumePlugin {
```

Comment:
Why a new probe function ? I think if we are going to split the probe function at all we should wait to see what other flags are gonna look like. It might not make sense to divide along the lines of recyclable and non-recyclable in the future.